### ISSUE TITLE
KeyError when re-running antismashed gbk file

### ISSUE THREAD

**Issue by zdk123** [antismash/antismash#207]
**Describe the bug**
When running antismash 5 in a docker container, I'm getting a KeyError when the `update_prediction` function from the nrps_pks/parsers module is called.

**Expected behavior**
I expect AD/KS domains to be sanitized and rebuilt from scratch since `--skip-sanitisation` is not flagged.

**To Reproduce**
```sh
wget https://mibig.secondarymetabolites.org/repository/BGC0000001/BGC0000001.1.region001.gbk
mkdir BGC0000001

## Gives error
docker run -v ${PWD}:/input/ -v ${PWD}/BGC0000001:/output/ --detach=false --rm \
    --user=$(id -u):$(id -g) antismash/standalone:5.0.0 BGC0000001.1.region001.gbk

## No error (but is the source of BGC0000001) 
efetch -db nuccore -format gb -id JF752342.1 > JF752342.1.gbk
docker run -v ${PWD}:/input/ -v ${PWD}/BGC0000001:/output/ --detach=false --rm \
    --user=$(id -u):$(id -g) antismash/standalone:5.0.0 JF752342.1.gbk
```

**System (please complete the following information):**
 - antismash/standalone:5.0.0

**Additional context**
Normally, I would think about just stripping out the annotations, but this is also interacting with issue #180, where we want to combine / reuse existing results.

thanks!

**Comment by SJShaw** [comment]
This was fixed in #171 and should be in the 5.1 release.

**Comment by zdk123** [comment]
thanks! I guess I missed that when searching for duplicate issues.
